### PR TITLE
Update Cosmos Metrics

### PR DESCRIPTION
**Pull request by robert-zaremba**
Cosmos ecosystem has EVM module and there are native assets from the get go. 
In fact, every cosmos chain token is a native assets from the bank module, and many chains represents their tokens using the bank module coin functionality.

Moreover, Cosmos should be seen as a technology and an ecosystem, rather than a Hub (although the Hub, with it's native token Atom, has the biggest market capitalization).

As technology, Cosmos SDK and the Tendermint consensus support block time and block finality of 1s (see Injective Protocol or DyDx v4 as an example). 
So, should we update the block time from 6s to 1s as well?

### CODE REVIEW

File: public/blockchains.json [hunk 1]
```diff
@@ -271,13 +271,13 @@
     "Tps": 10000,
     "# nodes": 150,
     "TX fees (USD)": 0.01,
-    "Block Time": "7s",
-    "Block Finalizing": "7s",
+    "Block Time": "6s",
+    "Block Finalizing": "6s",
```
Comment (robert-zaremba) [inline public/blockchains.json:275]: As technology, Cosmos SDK and the Tendermint consensus support block time and block finality of 1s (see Injective Protocol or DyDx v4 as an example).
So, should we update the block time from 6s to 1s as well?

File: public/blockchains.json [hunk 2]
```diff
@@ -271,16 +271,16 @@
...
     "Consense": "PoS-BFT",
-    "Native Assets": "no",
+    "Native Assets": "yes",
     "Smart Contract": "yes",
-    "EVM": "no",
+    "EVM": "yes",
     "WASM": "yes",
     "Other": "",
-    "Language": "Go",
+    "Language": "Go, Rust, Solidity",
```
Comment (robert-zaremba) [inline public/blockchains.json:283]: SDK modules are written in Go, but smart contracts can be written in Rust or Solidity.

**Review by danicuki — APPROVED** [review]

File: public/blockchains.json (hunk 1 above)
Comment (danicuki) [reply public/blockchains.json:275]: Even if it supports, I believe that if the main blockchain still uses the 6s block time, we should keep it on the `6s` value. But I understand that in some way we should bring these details to the end user. Do you have any suggestion about how we could improve the comparison table project as a whole?

Comment (robert-zaremba) [reply public/blockchains.json:275]: There is no such thing as "main blockchain". Cosmos is a an ecosystem. Atom is a currency of Gaia / Cosmos Hub.
`6s` is the default for Cosmos SDK chains, but it's very easy to change it, and more and more new chains are using smaller values.

**Pull request merged**